tools/llvm-kompile-compute-loc/main.cpp

```diff
@@ -60,17 +48,44 @@ get_kore_location(std::string &definition, int const &ordinal) {
   return std::nullopt;
 }
 
+std::optional<std::pair<std::string, int64_t>>
+get_k_location(std::string &definition, int const &ordinal) {
+  // Parse the definition.kore to get the AST.
+  kllvm::parser::kore_parser parser(definition);
+  auto kore_ast = parser.definition();
+  kore_ast->preprocess();
+
+  auto axiom = kore_ast->get_axiom_by_ordinal(ordinal);
+  auto result = get_start_line_location(axiom);
+  if (result) {
+    return result;
+  }
+  auto result2 = get_kore_location(definition, ordinal);
+  if (result2) {
+    return std::make_pair<std::string, uint64_t>("", *result2);
+  }
+  return std::nullopt;
```

Comment:
I think the function's implementation changed enough to justify a new name, perhaps something like `get_k_or_kore_location` or just `get_location`.

To be more explicit, I would also rename the result variable names to `k_loc` and `kore_loc`.